### ISSUE TITLE
Dont compile after update to 1.2.0 

### ISSUE THREAD

**Issue by kaleu50** [ckeditor/ckeditor5-angular#169]
I upgraded to version 1.2.0 and now no longer compiles my project

```
ERROR in ./node_modules/@ckeditor/ckeditor5-angular/fesm5/ckeditor-ckeditor5-angular.js
Module not found: Error: Can't resolve '@ckeditor/ckeditor5-watchdog/src/editorwatchdog' 
in 'C:\Users\giova\Documents\Guidlle\WebApplication\node_modules\@ckeditor\ckeditor5-angular\fesm5'
```

Angular Version: 7.2.16
CLI Version: 8.3,20

**Comment by givan2code** [comment]
Right there with you...

**Comment by JSCarter** [comment]
Same, I am on angular 9.0.4.

**Comment by ma2ciek** [comment]
Sorry guys, I'm on it. Looks like the import has an impact on the built package :/ Please, pin to the 1.1.2 version for a moment or just add the `@ckeditor/ckeditor5-watchdog@17` package to your dependencies.

**Comment by ma2ciek** [comment]
I've released the `1.2.1` version with a fix. Sorry guys for the inconvenience :/
